src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -50,9 +50,16 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
```

Comment:
can have 2 separate tests? parseCommand_deleteByIndex() and parseCommand_deleteByKeyword()?, easier to test and maintain also.